packages/amplication-git-utils/src/git/git.service.ts

```diff
@@ -144,6 +154,34 @@ export class GitClientService {
     throw new InvalidPullRequestMode();
   }
 
+  private async preCommitProcess({
+    gitClient,
+    branchName,
+    owner,
+    repositoryName,
+  }: PreCommitProcessArgs): PreCommitProcessResult {
+    await gitClient.git.checkout(branchName);
+    await gitClient.resetState();
+
+    const commitsList = await this.provider.getCurrentUserCommitList({
+      branchName,
+      owner,
+      repositoryName,
+    });
+
+    const latestCommit = commitsList[0];
+
+    if (!latestCommit) {
+      throw new Error(
+        "Didn't found a commit that has been created by amplication"
+      );
```

Comment:
```suggestion
      throw new Error(
        "Didn't find a commit that has been created by Amplication"
      );
```

Reply:
maybe use this error message as a const because we use it in more than one place